### ISSUE TITLE
Add result visualization to Colabs

### ISSUE THREAD

**Issue by gkielian** [ReaLLMASIC/nanoGPT#71]
Adding visualization examples for will essentially open up development via Colabs, and make sharing graphs of results and means to reproduce these results instantaneously accessible.

Notes:
- [train.py](http://train.py/) creates csv files in a csv_logs directory by default

Things to visualize:

Set 1: Training and Validation Losses
- training loss vs time
- validation loss vs time
- best (aka lowest) validation loss of a run from `run_experiments.py`

Set 2: Softmax Statistics
- beta/gamma values for Constantmax
- histogram of denominators of Softermax
